stdlib/email/src/main/ballerina/src/email/commons.bal

```diff
@@ -33,6 +34,7 @@ public type Email record {|
     string[] bcc?;
     string subject;
     string body;
```

Comment:
Do we need to change the type of message body as well along with the `contentType` support?
User may need to pass `json`, `xml` in email body

Reply:
Yes. Therefore, made it possible to use `json` and `xml` types as well.